js/views/errorScreen.js

```diff
@@ -13,9 +13,20 @@ var ErrorScreen = React.createClass({
   render: function() {
     var errorTitle, errorDescription, errorAction;
     if (CONSTANTS.ERROR_MESSAGE.hasOwnProperty(this.props.errorCode.code)){
+
+      var errorAction = CONSTANTS.SKIN_TEXT.ERROR_ACTION;
+      if( CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].action ){
+        errorAction = CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].action;
+      }
+
       errorTitle = Utils.getLocalizedString(this.props.language, CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].title, this.props.localizableStrings)
       errorDescription = Utils.getLocalizedString(this.props.language, CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].description, this.props.localizableStrings);
-      errorAction = Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.ERROR_ACTION, this.props.localizableStrings);
+      /// TODO - need to make countdown functionality display for all languages
+      if (this.props.errorCode.code === "future" && this.props.language === "en"){
```

Comment:
Can we make "future" a constant? We should consider making "en" a constant if it isn't already one as well.

Reply:
Added both